### PR TITLE
add predecessor run id to transcriptic launch command

### PR DESCRIPTION
**Pull request by aaronstrateos**
If predecessor id passed to launch command, it will print out the link to the predecessor run only if the run is successfully launched. 

@joshhacksthings If you don't want it to print the preceding run to stdout, I can remove it.  Please direct me to an example where I can set an attribute on the run object because I don't see it in the launch method. Can I just add a predecessor_id field to the payload in submit_launch_request()?

### CODE REVIEW

**Review by samjoch — DISMISSED** [review]

File: transcriptic/config.py
Comment (xiliuStrateos) [inline transcriptic/config.py:726]: I think the `predecessor_id` can be added to the `payload` directly. There if a filter step after it, it will filter out the None value items anyway.

**Review by xiliuStrateos — CHANGES_REQUESTED** [review]

**Comment by xiliuStrateos** [comment]
@joshhacksthings This PR lgtm, do you have any other comments?

File: transcriptic/config.py
Comment (joshhacksthings) [reply transcriptic/config.py:726]: agreed with Xi here, good catch!

**Comment by joshhacksthings** [comment]
@aaronstrateos you're good to merge once you address Xi's comment above! No need for my approval

**Review by xiliuStrateos — DISMISSED** [review]
lgtm! fix build

**Review by xiliuStrateos — APPROVED** [review]

**Pull request merged**
